cartoframes/utils/metrics.py

```diff
@@ -69,15 +78,16 @@ def check_valid_metrics_uuid(metrics_config):
     return metrics_config is not None and is_uuid(metrics_config.get(UUID_KEY))
 
 
-def build_metrics_data(event_name, extra_metrics_data):
+def build_metrics_data(event_name, extra_metrics_data, server_domain_tld):
     metrics_data = {
         'event_version': EVENT_VERSION,
         'event_time': get_local_time(),
         'event_source': EVENT_SOURCE,
         'event_name': event_name,
         'source_version': __version__,
         'installation_id': get_metrics_uuid(),
-        'runtime_env': get_runtime_env()
+        'runtime_env': get_runtime_env(),
+        'api_used': CLOUD_API if server_domain_tld in [PROD_DOMAIN_TLD, STAG_DOMAIN_TLD] else CUSTOM_API
```

Comment:
I would move this logic to `utils.get_api_used(server_domain_tld)` or `get_api_used(server_domain_tld)` (this file)

Reply:
:raised_hands: